docs/090_roadmap/README.md

```diff
@@ -1,5 +1,53 @@
 # Roadmap for Pepr
 
+_2025 Roadmap_  
+
+## Phase 1: Code Quality - Experimentation
+
+- **Q1**:
+  - **Turn on eslint enforcement and configure settings and see no warnings**:
+    - Eliminate circular dependencies, complexity, return statements, etc.
+  - **Metric and Performance Baselining**:
+    - Establish a baseline for performance and resource utilization metrics. Use this data to make informed decisions about the direction of the project in terms of Deno2
+  - **Nightly Release**:
+    - Establish a nightly release process. This will help us to catch bugs early and ensure that the project is always in a releasable state.
+
+## Phase 2: Durable Storage for Metrics and Performance Tests / Transactional Pepr Store
+
+- **Q2**:
+  - **Professional Dashboard displaying metrics and performance tests originating from CI**:
+  - **Determine if a Transactional PeprStore makes sense**:
+    - Sus out details involved with having a transactional Pepr Store. What are the implications of this? What are the benefits? What are the drawbacks? What are the use-cases? What are the technologies that can be used to implement this?
```

Comment:
Based on a [comment](https://github.com/defenseunicorns/pepr/issues/1432#issuecomment-2694041792) by @slaskawi this may now not be a priority from a UDS perspective as they will not rely on the store for Keycloak integration. Lets keep an eye on this

Reply:
Moved to deprioritized work